src/_guides/language/concurrency/index.md

```diff
@@ -324,6 +314,18 @@ setting up and managing worker isolates:
 
 [`Isolate.run()`]: {{site.dart-api}}/dev/dart-isolate/Isolate/run.html
 
+{{site.alert.flutter-note}}
+  If you're using Flutter,
+  consider using [Flutter's `compute()` function][]
+  instead of `Isolate.run()` to allow your code to work
+  on both [native and non-native platforms][].
+  `Isolate.run()` is the more ergonomic API choice
+  for Flutter apps targeting native platforms only. 
```

Comment:
**suggestion:** This is fine as is, but I would suggest making this more concise.
```suggestion
  If you're using Flutter,
  consider using [Flutter's `compute()` function][]
  instead of `Isolate.run()`. The `compute` function
  allows your code to work  on both
  [native and non-native platforms][].
  Use `Isolate.run()` when targeting native platforms only.
  This Isolate function serves as a more ergonomic
  API choice for native Flutter apps. 
```